instrumentation/google-http-client-1.19/src/main/java/io/opentelemetry/auto/instrumentation/googlehttpclient/GoogleHttpClientInstrumentation.java

```diff
@@ -143,7 +142,6 @@ public static void methodExit(
           // for a failed request.  Thus, check the response code
           if (response != null && !response.isSuccessStatusCode()) {
             span.setStatus(Status.UNKNOWN);
-            span.setAttribute(MoreAttributes.ERROR_MSG, response.getStatusMessage());
```

Comment:
Why was this just removed?

Reply:
This is setting an HTTP response as an error tag, but our new convention is only about exceptions, not HTTP status line. Keeping this would be confusing wrt to our other tags.